.golangci.yml

```diff
@@ -0,0 +1,90 @@
+version: "2"
+linters:
+  enable:
+    # Commented entries here are not enabled by default, but that we should
+    # add once the relevant lines are fixed up.
+    
+    # - errorlint # found 2 cases to be fixed.
+    # - exhaustive # found some cases that should be fixed.
+    # - exhaustruct # found some structs which missed fields in initializing.
+    - govet
```

Comment:
Do we want to explicitly list linters which are enabled by default (https://golangci-lint.run/usage/linters)? If so, you should mention [ineffassign](https://golangci-lint.run/usage/linters#ineffassign) too.

Reply:
Dropped the enabled by default that work.